resources/scripts/editor/components/block-components.js

```diff
@@ -187,6 +195,23 @@ export default function BlockComponents({attributes, components, onChange, props
             </MediaUploadCheck>
           </BaseControl>
         );
+      case 'Link':
```

Comment:
I suggest 

```
case 'Link':
        return (
          <BaseControl
            key={component.name}
            label={component.label}
          >
            <LinkControl
              key={component.name}
              label={component.label}
              help={component.help}
              value={item ? item[component.name] : attributes[component.name]}
              onChange={(value) => onChange(id, component.name, value)}
              onRemove={() => onChange(id, component.name, {})}
            />
          </BaseControl>
        );
```

Reply:
I used that method because of two things:
- I wanted not to show the link title if it is used strictly as a link field
- for some reason LinkControl component doesn't update the title if you change the link in that field - that would be confusing for the end user